blocks/variables_dynamic.ts

```diff
@@ -156,6 +157,10 @@ const CUSTOM_CONTEXT_MENU_VARIABLE_GETTER_SETTER_MIXIN = {
     } else {
       this.getInput('VALUE')!.connection!.setCheck(variableModel.type);
     }
+
+    if (this.rendered) {
+      (this as BlockSvg).queueRender();
+    }
```

Comment:
Instead of having this inside the individual block definition, you can override `setCheck` in the [rendered connection](https://github.com/google/blockly/blob/develop/core/rendered_connection.ts) class, and call `queueRender` there! This way whenever a connection check gets updated on any block, we'll be sure the shapes of the connections get updated =)

Reply:
Thank you for taking the time to review my PR and providing your feedback. I appreciate your suggestions. updated